### PR TITLE
Update Timeout BBE syntax to beta1

### PR DESCRIPTION
**Pull request by shafreenAnfar**
## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

File: examples/http-timeout/http_timeout.bal [hunk 1]
```diff
@@ -1,46 +1,28 @@
...
 service /timeout on new http:Listener(9090) {
 
-    resource function get .(http:Caller caller, http:Request request) {
-        http:Response|error backendResponse =
-            backendClientEP->forward("/hello", request);
+    resource function get .() returns http:InternalServerError|string|error? {
+        string|error backendResponse =
+                        backendClientEP->get("/hello");
 
         // If `backendResponse` is an `http:Response`, it is sent back to the
```
Comment (chamil321) [inline examples/http-timeout/http_timeout.bal:16]: We may need to change the doc hear we take the payload instead of the `http:Response`

Comment (shafreenAnfar) [reply examples/http-timeout/http_timeout.bal]: Fixed it.

**Review by chamil321 — APPROVED** [review]

File: examples/http-timeout/http_timeout.bal [hunk 2]
```diff
@@ -1,46 +1,28 @@
...
-            response.statusCode = http:STATUS_INTERNAL_SERVER_ERROR;
-            string errorMessage = backendResponse.message();
-            string expectedMessage = "Idle timeout triggered before " +
-                "initiating inbound response";
-            if (errorMessage == expectedMessage) {
-                response.setPayload(
-                    "Request timed out. Please try again in sometime."
-                );
+            if (backendResponse is http:IdleTimeoutError) {
+                return { body: "Request timed out. Please try again in sometime."};
```
Comment (chamil321) [inline examples/http-timeout/http_timeout.bal:23]: This line seems to be exceeding the column length. 
```
[/examples/http-timeout/http_timeout.bal] Line number 23.
```

Comment (shafreenAnfar) [reply examples/http-timeout/http_timeout.bal]: Fixed it.

**Pull request merged**